### PR TITLE
[WNMGDS 1166] Add lightweight alert

### PR DESCRIPTION
**Pull request by line47**
## Summary
Adding a lightweight alert weight option to our alerts

### Added
- Added a new prop `weight` to the alert with the current only option being `"lightweight"`.
- Added new styling to set the background color to white for the lightweight alerts

### Fixed
List any bug fixes.

## How to test
- Run the doc site locally with `yarn start`
- Add `weight="lightweight"` to an alert component and see that the alert gets the lightweight styling with a color left border and white background
- Demo URL http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1166-lightweight-alert/components/alert/

### CODE REVIEW

File: packages/design-system/src/styles/components/_Alert.scss [hunk 1]
```diff
@@ -76,6 +76,9 @@ $alert-icon-size: $spacer-5;
       margin-top: 0;
     }
   }
+  &.ds-c-alert--lightweight {
```
Comment (scheul93) [inline packages/design-system/src/styles/components/_Alert.scss:79]: [question] Does this class need extra specificity? Is that why the selector needs to be `.ds-c-alert.ds-c-alert--lightweight`?
If not, consider moving the `ds-c-alert--lightweight` selector to below/outside the `.ds-c-alert` block scope. It removes one degree of specificity and allows easier overwriting down stream

File: packages/design-system/src/components/Alert/Alert.test.tsx [hunk 2]
```diff
@@ -37,6 +37,12 @@ describe('Alert', function () {
     expect(wrapper.hasClass('ds-c-alert--error')).toBe(true);
   });
 
+  it('appears as an lightweight alert', () => {
```
Comment (scheul93) [inline packages/design-system/src/components/Alert/Alert.test.tsx:40]: 🎉 A test! 🎉

**Review by pwolfert — APPROVED** [review]
Nice work! I tried it with all the variations, and it looks good.

File: packages/design-system/src/styles/components/_Alert.scss (hunk 1 above)
Comment (pwolfert) [inline packages/design-system/src/styles/components/_Alert.scss:79]: Only thing I might change is putting a line break above this

**Comment by scheul93** [comment]
Can an alert be both "lightweight" & "error" or "lightweight" & "success"?

File: packages/design-system/src/styles/components/_Alert.scss (hunk 1 above)
Comment (scheul93) [reply packages/design-system/src/styles/components/_Alert.scss:79]: Nevermind. I think I found my answer below AND why the lightweight needed extra specificity 🙃

**Review by scheul93 — APPROVED** [review]

**Pull request merged**